@here/harp-mapview/lib/DataSource.ts

```diff
@@ -50,6 +50,12 @@ export abstract class DataSource extends THREE.EventDispatcher {
      */
     addGroundPlane: boolean = false;
 
+    /**
+     * Boolean which says whether a [[DataSource]] produces tiles that fully cover the tile, i.e.
+     * tiles underneath are completely hidden.
+     */
+    isFullyCovering: boolean = false;
```

Comment:
This should be a method that could be overridden and the default implementation should just return `addGroundPlane`